nvflare/apis/utils/decomposers/flare_decomposers.py

```diff
@@ -111,7 +111,7 @@ def supported_type():
         return Workspace
 
     def decompose(self, target: Workspace) -> Any:
-        return [target.root_dir, target.name, target.config_folder]
+        return [target.root_dir, target.site_name, target.config_folder]
```

Comment:
Note this change is the same as in #914

Reply:
that's fine. PR merge will take care of this.